bigquery/nox.py

```diff
@@ -88,6 +88,39 @@ def system_tests(session, python_version):
 
 
 @nox.session
+@nox.parametrize('python_version', ['2.7', '3.6'])
+def snippets_tests(session, python_version):
+    """Run the system test suite."""
+
+    # Sanity check: Only run system tests if the environment variable is set.
+    if not os.environ.get('GOOGLE_APPLICATION_CREDENTIALS', ''):
+        session.skip('Credentials must be set via environment variable.')
+
+    # Run the system tests against latest Python 2 and Python 3 only.
+    session.interpreter = 'python{}'.format(python_version)
+
+    # Set the virtualenv dirname.
+    session.virtualenv_dirname = 'snip-' + python_version
+
+    # Install all test dependencies, then install this package into the
+    # virtualenv's dist-packages.
+    session.install('mock', 'pytest', *LOCAL_DEPS)
+    session.install(
+        os.path.join('..', 'storage'),
+        os.path.join('..', 'test_utils'),
+    )
+    session.install('.')
```

Comment:
I noticed that in the system tests, we have this instead of
```
session.install('-e', '.')
```
The advantage of the latter is that you can fix your files and re-run with nox -r, so the virtualbox doesn't need to be re-installed. Is there a reason not to use -e?

Reply:
You're right. I don't think there is a good reason to use install `.` instead of `-e .`. I wondered why I'd get out-of-date results with `-r`. This is probably the reason.